client/me/profile-links-add-other/index.jsx

```diff
@@ -59,6 +58,24 @@ const ProfileLinksAddOther = createReactClass( {
 		return false;
 	},
 
+	recordClickEvent( action ) {
+		this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
+	},
+
+	getClickHandler( action ) {
+		return () => this.recordClickEvent( action );
+	},
+
+	getFocusHandler( action ) {
+		return () => this.props.recordGoogleEvent( 'Me', 'Focused on ' + action );
+	},
+
+	handleCancelButtonClick( event ) {
+		event.preventDefault();
```

Comment:
Do we really need this here? The button doesn't have an `href` or anything, does it?

Reply:
Yes, because the cancel button is essentially a `<button type="submit" />` too. We can, of course, change that easily, but I didn't want to introduce any additional changes here, because it would require an additional set of testing steps. Good call though, I'd say it's worth a revisit in another PR.